.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,66 @@
+<!-- 
+  Thank you very much for contributing to Apache HugeGraph, we are happy that you want to help us improve it!
+
+  Here are some tips for you:
+    1. If this is your first time, please read the [contributing guidelines](https://github.com/apache/hugegraph/blob/master/CONTRIBUTING.md)
+
+    2. If a PR fix/close an issue, type the message "close xxx" (xxx is the link of related 
+issue) in the content, GitHub will auto link it (Required)
+
+    3. Name the PR title in "Google Commit Format", start with "feat | fix | perf | refactor | doc | chore", 
+      such like: "feat(core): support the PageRank algorithm" or "fix: wrong break in the compute loop" (module is optional)
+      skip it if you are unsure about which is the best component.
+
+    4. One PR address one issue, better not to mix up multiple issues.
+
+    5. Put an `x` in the `[ ]` to mark the item as CHECKED. `[x]` (or click it directly after 
+published)
+-->
+
+## Purpose of the PR
+
+- close #xxx  <!-- or use "fix #xxx", "xxx" is the ID-link of related issue, e.g: close #1024 -->
+
+<!--
+Please explain more context in this section, clarify why the changes are needed. 
+e.g:
+- If you propose a new API, clarify the use case for a new API.
+- If you fix a bug, you can clarify why it is a bug, and should be associated with an issue.
+-->
+
+## Main Changes
+
+<!-- Please clarify what changes you are proposing. The purpose of this section is to outline the changes and how this PR fixes the issue. These change logs are helpful for better ant faster reviews.)
+
+For example:
+
+- If you introduce a new feature, please show detailed design here or add the link of design documentation.
+- If you refactor some codes with changing classes, showing the class hierarchy will help reviewers.
+- If there is a discussion in the mailing list, please add the link. -->
+
+## Verifying these changes
+
+<!-- Please pick the proper options below -->
+
+- [ ] Trivial rework / code cleanup without any test coverage. (No Need)
+- [ ] Already covered by existing tests, such as *(please modify tests here)*.
+- [ ] Need tests and can be verified as follows:
+    - xxx
```

Comment:
keep "None"?

Reply:
I don't quite understand, could you explain in more detail? :)

Reply:
mean that we prefer to replace "xxx" with "None" since we may not need to test manually and modify the default list.

Reply:
> mean that we prefer to replace "xxx" with "None" since we may not need to test manually and modify the default list.

`None` seems duplicate with the first option (No Need), and devs could remove it manually, we could test & enhance it later (listen the users feedback)